src/main/java/teammates/storage/sqlentity/Instructor.java

```diff
@@ -84,6 +87,17 @@ public void setPrivileges(InstructorPrivileges instructorPrivileges) {
         this.privileges = instructorPrivileges;
     }
 
+    /**
+     * Gets the archived status of the instructor.
+     */
+    public boolean getIsArchived() {
+        return isArchived;
+    }
+
+    public void setIsArchived(boolean isArchived) {
+        this.isArchived = isArchived;
+    }
```

Comment:
We can remove this. Will not be necessary anymore after removal of archive course feature.

Reply:
Yup!